docs/adr/0007-backup-policy.md

```diff
@@ -0,0 +1,23 @@
+# 7. Retain database backup for 90 days
+
+Date: 2023-02-08
+
+## Status
+
+Accepted
+
+## Context
+
+The database for job-server is currently stored on a managed cluster which is able to backups for seven days. A recent regression in the codebase caused some database fields to be truncated, and was not spotted until about six days after deployment. In this case, we were able to repair the truncated data using an old backup, but if it had gone a little bit longer without being spotted we would have experienced minor data loss.
+
+Our current opinion is that although seven days is sufficiently long enough to spot recover in the event of catastrophic dataloss, we should keep backups for longer to help recover from more subtle dataloss/degradation.
+
+The data stored in the database is not considered sensitive, and because of our desire & policy for openness a significant proportion of the contents are (or will become) public information.
```

Comment:
I'm not sure about this sentence. We will be keeping usernames and passwords in that database, which is sensitive information.

Reply:
We also have user data from the applications form which would stay private, and OAuth tokens&data from GitHub.  However, I do see what Tom is getting at here, and we have generally wanted to make everything apart from user-specific data publich over time.  Myabe this isn't the best place to record that though?

Reply:
sorry, yes, this is condensed from two ideas & doesn't quite make sense on its own. Because of the question-mark about storing the backup in Frankfurt, we had two different ideas about sensitivity - one is "user identifying information to which data protection rules may apply" (e.g. private data about an individual) and one is "internal secrets that we need to protect" (e.g. keys/tokens). I'm talking about the former here, and attempting clumsily to note that although we store usernames, we also publically publish the name of the author & link to their github commits with their public github username.